### ISSUE TITLE
widget is off center and partly cut off

### ISSUE THREAD

**Issue by nutpantz** [ukanth/afwall#424]
I have a mega i527m running the latest nova launcher  And the latest afwall donate
When using the widget it is off center to the right
And about 20% is cut of making it partly unusable.
If you need any more information you can contact me at my email

**Comment by CHEF-KOCH** [comment]
We not changed the widgets, so this is an launcher problem. Please do not use cracked apps.
I'm also on Nova Launcher Prime and it works. 

From the changelog:
```
Nova 4.1.0-beta3
Bug fixes
Optimizations
Nova 4.1.0-beta2
Fix FC on Android 4.x
Nova 4.1.0-beta1
Normalize Icon Size
Marshmallow, and Marshmallow grayscale style search bar
Runtime permissions on Marshmallow
Optimizations and bug fixes
```

Maybe an upgrade to > Nova 4.1.0-beta1 resolve the problem.

**Comment by nutpantz** [comment]
I have downloaded from Google play the newest Nova launcher, and apex launcher. I uninstalled and reinstalled my nova prime from play store and I checked with the bundled launcher of the Rom.
All three have the same issue. I flashed factory oem 442 for the mega 2 and installed nova launcher and prime and it is only a little off.
So it must be an issue with afwall primary with cm,  and the mega6.3. I'll upload photos to xda to show the different launchers.
